build/style-compiler/LessAggregation.cs

```diff
@@ -210,6 +210,9 @@ public static IEnumerable<Item> EnumerateAllItems(string sourcePath, string dist
 
                     foreach (var colorSchemeName in theme.ColorSchemeNames)
                     {
+                        if (distribution.ExcludedColorSchemes != null && Array.IndexOf(distribution.ExcludedColorSchemes, colorSchemeName) >= 0)
```

Comment:
Oh this is so JavaScripty 🍬 
`distribution.ExcludedColorSchemes.Contains(colorSchemeName)`